src/components/Editor/EditPublications.vue

```diff
@@ -166,10 +163,18 @@
     </v-card-text>
     <v-card-actions>
       <v-btn
-        class="primary"
-        @click="updateRecordPub()"
+          class="primary"
+          :loading="loading"
+          @click="saveRecord(false)"
+      >
+        Save and continue
+      </v-btn>
+      <v-btn
+          :loading="loading"
+          class="primary"
+          @click="saveRecord(true)"
```

Comment:
here we have another eslint warning. 
![image](https://user-images.githubusercontent.com/25683981/105480287-6f307680-5c9d-11eb-9818-5cb010e93ba1.png)